### ISSUE TITLE
www.handelsblatt.com is blocked (Access denied)

### ISSUE THREAD

**Issue by oc34nz** [uBlockOrigin/uAssets#331]
Hi all,

Using uBlockOrigin (v_1.11.4) in Firefox (v_52.0.1 - 32bit), 
Access to the Website www.handelsblatt.com is denied. 

The Website prompts the Visitor to deactivate its Adblock Software, otherwise Access to the Site's content is denied. See attached printscreen.

![www handelsblatt com](https://cloud.githubusercontent.com/assets/26655174/24331301/c6510138-1231-11e7-9614-35f7f5f3b7fb.png)

### Versions

- Browser/version: Firefox v_52.0.1 - 32bit
- uBlock Origin version: v_1.11.4

### Settings

I changed nothing within uBlockOrigin, all settings is on default. --> I'm new to uBlockOrigin, installed it yesterday (26.03.2017). Had no time for playing with settings :-)

### Notes

Before 26.03.2017, I used another fork, uBlock, (https://addons.mozilla.org/en-US/firefox/addon/ublock/), which resulted in many of such "Site Access Denial" issues. Now, by using your fork uBlockOrigin, all known issues is obsolet, at once. The only remaining issue (to date) is with www.handelsblatt.com. It still blocks Site Access, even when using uBlockOrigin. 

Happy Weekend to all,
Beat

**Comment by gorhill** [comment]
Does the following filter break important functionality on the site?

    ||handelsblatt.com/preparesite/empty.js$script,first-party

**Comment by oc34nz** [comment]
Whoopy, this works! 
 
After pasting this string into "My Filters" section,
 
www.handelsblatt.com is wonderfully unlocked. 
 
Sublinks, even for multimedia content, work good.
 
Will try this string with other Sites (if found), before starting another issue.
 
Thank you so much :-)
 
Beat
 
Von: Raymond Hill [mailto:notifications@github.com] 
Gesendet: Sonntag, 26. März 2017 15:09
An: uBlockOrigin/uAssets
Cc: oc34nz; Author
Betreff: Re: [uBlockOrigin/uAssets] www.handelsblatt.com is blocked (Access denied) (#331)
 
Does the following filter break important functionality on the site?
||handelsblatt.com/preparesite/empty.js$script,first-party

**Comment by gorhill** [comment]
Since it's a classic filter, you should report it to EasyList Germany maintainers: <https://forums.lanik.us/viewforum.php?f=90>.

**Comment by okiehsch** [comment]
@gorhill I think you should consider that `EasylistGermany` seems hesitant, atleast in my opinion,
to add filters for big german sites, who use anti-adblock techniques such as `handelsblatt.com`.
Examples:
1. `handelsblatt.com` working filter suggested  Nov 19, 2016
`https://forums.lanik.us/viewtopic.php?p=107968#p107968`
Filter still works, but has not been added yet.
2. `sport1.de` working filter suggested Jun 05, 2016
`https://forums.lanik.us/viewtopic.php?p=95397#p95397`
Filter still works, but has not been added yet. Fixed with `uBO-filter`
3. `sueddeutsche.de` working filter suggested  Dec 09, 2016
`https://forums.lanik.us/viewtopic.php?p=109676#p109676`
Filter still works, but has not been added yet.  Fixed with `uBO-filter`

Then there is the case of `bild.de` which actually threatened legal action, if a filter would appear
that would circumvent `BILdSmart` and so far no working filter for `bild.de` has appeared in
`EasylistGermany`.
